commit 59dc6d61b0ad4a7e91cdda076169cd328f58a0c8
Author: Rico Tzschichholz <ricotz@ubuntu.com>
Date:   Sat Aug 25 17:20:23 2018 +0200

    Adapt to type change of Gdk.Key constants to fix build with valac 0.42

diff --git src/Window.vala src/Window.vala
index 6213661..cffb559 100644
--- src/Window.vala
+++ src/Window.vala
@@ -225,7 +225,11 @@ public class Audience.Window : Gtk.Window {
 
     /** Returns true if the code parameter matches the keycode of the keyval parameter for
     * any keyboard group or level (in order to allow for non-QWERTY keyboards) **/
+#if VALA_0_42
+    public bool match_keycode (uint keyval, uint code) {
+#else
     public bool match_keycode (int keyval, uint code) {
+#endif
         Gdk.KeymapKey [] keys;
         Gdk.Keymap keymap = Gdk.Keymap.get_for_display (Gdk.Display.get_default ());
         if (keymap.get_entries_for_keyval (keyval, out keys)) {
